indigo/distribution.yaml

```diff
@@ -6172,6 +6172,21 @@ repositories:
       url: https://github.com/fkie/multimaster_fkie.git
       version: indigo-devel
     status: maintained
+  multirobot_tracker:
+    doc:
+      type: git
+      url: https://github.com/multirobot-ferr/multirobot_tracker.git
+      version: indigo-devel
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/multirobot-ferr/multirobot_tracker.git
```

Comment:
For releases you should use a separate release repo. You will need to use [bloom](http://wiki.ros.org/bloom) to populate that repo. `bloom` will then also take care and create a PR to add this `release` entry for you.